parts/k8s/kubernetescustomscript.sh

```diff
@@ -151,4 +136,14 @@ if $REBOOTREQUIRED; then
   /bin/bash -c "shutdown -r 1 &"
 else
   runAptDaily &
+fi
+
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
```

Comment:
similar to last comment, probably make L131 always true would make future re-enable & merge easier.